Jenkinsfile.download_logs

```diff
@@ -5,7 +5,7 @@ pipeline {
         string(name: 'REMOTE_SERVICE_URL', defaultValue: 'https://api.openshift.com', description: 'Service URL')
     }
 
-    triggers { cron('H/10 * * * *') }
+    triggers { cron('H/30 * * * *') }
```

Comment:
why?

Reply:
Now each run might take longer. So together with @ErezAlster we figured out we should extend the interval as well.